sync/api/projects.py

```diff
@@ -102,6 +102,7 @@ def update_project(
     project_id: str,
     description: str = None,
     s3_url: str = None,
+    app_id: str = None,
```

Comment:
I find it useful to be able to set the external ID so that the same ID can be used across environments